f90wrap/scripts/f2py_f90wrap.py

```diff
@@ -51,7 +51,8 @@
 def main():
 
     import numpy
-    if not tuple([int(x) for x in numpy.__version__.split('.')[0:2]]) >= (1,3):
+    numpy_version = tuple([int(x) for x in numpy.__version__.split('.')[0:2]])
+    if not numpy_version >= (1,3):
        raise ImportError('f2py-f90wrap tested with numpy version 1.3 or later, found version %s' % numpy.__version__)
```

Comment:
We should change this to numpy v1.13 for consistency with the version bound in pyproject.toml

Reply:
Yes, should be good now !